doc/man7/OSSL_PROVIDER-serialization.pod

```diff
@@ -0,0 +1,99 @@
+=pod
+
+=head1 NAME
+
+OSSL_PROVIDER-serialization - OpenSSL serialization provider
+
+=head1 DESCRIPTION
+
+The OpenSSL serialization provider supplies the serialization for OpenSSL's
+asymmetric cryptography.
+
+=head2 Properties
+
+The implementations in this provider specifically have this property
+defined:
+
+=over 4
+
+=item "provider=serialization"
+
+=back
+
+It may be used in a property query string with fetching functions.
+
+It isn't mandatory to query for this property, except to make sure to get
+implementations of this provider and none other.
+
+The serialization algorithms in the default provider also have
+provider=serialization set.
```

Comment:
Hmmmm. Really? This seems wrong (and doesn't seem to match what is implemented).

Reply:
Oops....